src/common/components/select-repository-search/index.js

```diff
@@ -0,0 +1,29 @@
+import React, { Component, PropTypes } from 'react';
+import styles from './search.css';
+
+export default class SelectRepositorySearch extends Component {
+
+  render() {
+    let id = this.props.id || 'search-repositories';
+    let label = this.props.label || 'Search';
+    let placeholder = this.props.placeholder || 'Filter respositories...';
+
+    return (
+      <label htmlFor={ id }>
+        { label }
+        <input
```

Comment:
To make this input component functional later it would need a `value` and `onChange` handler bindings.

We could simply pass them from props:

```js
<input
  value={props.value}
  onChange={props.onChange}
/>

// ...

SearchInput.propTypes = {
  // ...
  value: PropTypes.string.isRequired,
  onChange: PropTypes.func.isRequired
}
```